AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -1578,7 +1578,7 @@ produkte.erstegroup.com##.OverlayDisclaimer
 ||paih.gov.pl/_js/2021/jquery.magnific-popup.js
 evropochta.by#%#//scriptlet('set-local-storage-item', 'saveCookie', 'true')
 evropochta.by#%#//scriptlet('set-local-storage-item', 'otherCookie', 'false')
-telesign.com##.osano-cm-window__widget
+finalfantasyxiv.com,telesign.com##.osano-cm-window__widget
```

Comment:
I can't reproduce the cookie popup. It is blocked by `||cmp.osano.com^`

Reply:
i can reproduce this chrome

Reply:
Do you have AdGuard Annoyances filter enabled?

@Alex-302 @jellizaveta please take a look

Reply:
you're right. when i turned on annoyances, this widget disappeared. Should i discard this change?

Reply:
yes, please discard